main/auth/gae.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+
+from __future__ import absolute_import
```

Comment:
I think that one was needed because the filename was called `google.py` and were some conflicts with it.. if that's not the case we could get rid of it :)

(Note to self: read about it)